tools/codegen/api/python.py

```diff
@@ -303,7 +438,16 @@ def output_idx(self) -> int:
     # For a translation to mypy-valid type signatures, see
```

Comment:
The old comment above made me wonder whether we are going to close the gap between PythonArgParser schema syntax and gen_pyi schema syntax (since there seems to be legit reason to keep them different...)